weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/FeedbackGridInner.tsx

```diff
@@ -9,16 +13,21 @@ import {Feedback} from '../pages/wfReactInterface/traceServerClientTypes';
 import {StyledDataGrid} from '../StyledDataGrid';
 import {FeedbackGridActions} from './FeedbackGridActions';
 import {FeedbackTypeChip} from './FeedbackTypeChip';
-import {isHumanAnnotationType} from './StructuredFeedback/humanAnnotationTypes';
+import {
+  getHumanAnnotationNameFromFeedbackType,
+  isHumanAnnotationType,
+} from './StructuredFeedback/humanAnnotationTypes';
 
 type FeedbackGridInnerProps = {
   feedback: Feedback[];
   currentViewerId: string | null;
+  showAnnotationName: boolean | undefined;
```

Comment:
I think `showAnnotationName?: boolean` is more idiomatic

Reply:
oh, yes of course.